test_e2e_v2.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source common.sh
+
+fetch_tools
```

Comment:
We can restructure the 2 e2e tests scripts to avoid duplication (e.g. fetch_tools, build_kb, docker pull and kind load docker-image).

Reply:
👍

Reply:
👍

Reply:
Hi @mengqiy, 

I tried more than one approach to attend it and all fail. See that we are with the CI broken for a while which is blocking we move forward with another priority. Also, IMHO the script still requiring a good review/refactor. Let's do it piece by piece? I mean what do you think about we track an issue to refactory them, centralize them in the scripts dir an etc?